deps.ts

```diff
@@ -1,3 +1,3 @@
-export { decode, encode } from "https://deno.land/std@v0.35.0/strings/mod.ts";
-export { format as byteFormat } from "https://deno.land/x/bytes_formater@1.1.0/mod.ts";
-export { replaceParams } from "https://deno.land/x/sql_builder@1.3.5/util.ts";
+export { decode, encode } from "https://deno.land/std/strings/mod.ts"
+export { format as byteFormat } from "https://deno.land/x/bytes_formater/mod.ts"
+export { replaceParams } from "https://deno.land/x/sql_builder/util.ts"
```

Comment:
I disapprove of removing semver because upstream dependencies are very likely to make destructive changes that make them unusable